src/main/java/org/folio/innreach/external/client/feign/InventoryStorageClient.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.innreach.external.client.feign;
+
+import org.folio.innreach.domain.dto.folio.inventoryStorage.InstanceIterationRequest;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@FeignClient("inventory-storage")
+public interface InventoryStorageClient {
+
+  @PostMapping("/instance-storage/instances/iteration")
+  void startInitialContribution(@RequestBody InstanceIterationRequest request);
+}
```

Comment:
`/instance-storage` endpoints shouldn't contain `/inventory-storage` in the root. So lets change the client to fix path
```
@FeignClient("instance-storage")
public interface InstanceStorageClient {

  @PostMapping("/instances/iteration")
  void startInitialContribution(@RequestBody InstanceIterationRequest request);
}
```